src/test/java/com/redhat/exhort/providers/Python_Provider_Test.java

```diff
@@ -149,6 +149,42 @@ void test_the_provideStack_with_properties(String testFolder)
     assertThat(dropIgnored(new String(content.buffer))).isEqualTo(dropIgnored(expectedSbom));
   }
 
+  @ParameterizedTest
+  @MethodSource("testFolders")
+  void test_the_provideStack_with_pipdeptree(String testFolder)
+      throws IOException, InterruptedException {
+    // create temp file hosting our sut package.json
+    System.setProperty("EXHORT_PIP_USE_DEP_TREE", "true");
+    var tmpPythonModuleDir = Files.createTempDirectory("exhort_test_");
+    var tmpPythonFile = Files.createFile(tmpPythonModuleDir.resolve("requirements.txt"));
+    try (var is =
+        getResourceAsStreamDecision(
+            this.getClass(),
+            new String[] {"tst_manifests", "pip", testFolder, "requirements.txt"})) {
+      Files.write(tmpPythonFile, is.readAllBytes());
+    }
+    // load expected SBOM
+    String expectedSbom;
+    try (var is =
+        getResourceAsStreamDecision(
+            this.getClass(),
+            new String[] {"tst_manifests", "pip", testFolder, "expected_stack_sbom.json"})) {
+      expectedSbom = new String(is.readAllBytes());
+    }
+    // when providing stack content for our pom
+    var content = this.pythonPipProvider.provideStack(tmpPythonFile);
+    String pipdeptreeContent = this.getStringFromFile("tst_manifests", "pip", "pipdeptree.json");
+    String base64Pipdeptree = new String(Base64.getEncoder().encode(pipdeptreeContent.getBytes()));
+    System.setProperty("EXHORT_PIP_PIPDEPTREE", base64Pipdeptree);
```

Comment:
@JudeNiroshan Actually this one wasn't in exhort-javascript-api ( i relating both for the logic and test here) , but you did it exactly the same as `EXHORT_PIP_SHOW` and `EXHORT_PIP_SHOW` and it simplifies testing and also make it more easy to integrate it with pipelines frameworks like jenkins, tekton and github action, so i think it's a good decision from your side.